### ISSUE TITLE
urn:cts:greekLit:tlg5026.msA.GKolov:6.37 

### ISSUE THREAD

**Issue by GKolov** [homermultitext/scholia-english#249]
Lemma (leave blank if no lemma): οὐδε γὰρ οὐ Δρύαντος

Comment: Some note that he talks about the god Dionysos. This Lykourgos has the same name as that [found in the verse]  Lykourgos  who killed  him, and Druas [is the same] as Lapithos Dionysos, the son of Zeus and  Kybele [the goddess] of Phrygia; he was purified by Rea  and made the ceremonies,  he was totally prepared by the goddess  and he was turning all over the earth. Dancing and receiving honors he was ahead of all .When he came to  Thrace, Lykourgos, the son of Druas  grieved (?) Hera, blind from the hate (against Dionysos) expelled him from the earth and he attacks him and his feeders.  They happened to celebrate mysteries with him. He comes being accompanied by his feeders. He hastened to punish the god by the whip. He is hidden in the sea by fear and he was removed by  Thetis  and Eyrynome: Lykourgos didn't die  without pay, he had the punishment from the people; he was deprived of his eye in front of  Zeus, many [commentators] mentionned  this  story, firstly [the author who invented] Europia, that is Eumelos.

**Comment by caseydue** [comment]
Lemma (leave blank if no lemma): οὐδε γὰρ οὐ Δρύαντος

Comment: Some mark [the verse with a critical sign] because he talks about the god Dionysos. This Lykourgos has the same name as that [found in the verse] "the one whom Lykourgos killed" [= Iliad 7.142], and Dryas [is the same] as the Lapith. Dionysos, the son of Zeus and Semele, after being purified by Rhea in the Kybela [mountains] of Phrygia, and after arranging the rites and receiving all the equipment from the goddess he went throughout the whole earth dancing and obtaining worship he led the way before all mankind. When he came to Thrace, Lykourgos, the son of Dryas, tormenting him with the blind hatred of Hera, expelled him from the land and he attacks him and those nurturing him. For they happened to celebrate mysteries with him. Driven by a god-sent whip he hastened to exact vengeance on the god. Dionysos enters the sea in fear and he is taken up by Thetis and Eyrynome. And so Lykourgos not without paying for his impiety suffered punishment from the people; for he was deprived of his eye by Zeus. Many have made mention of this story, first and foremost the one who composed the Europia, that is, Eumelos.

[Note: In the 9th century manuscript known as Z, a similar version of this note has διδαχθεὶς ("having been taught") where this manuscript has διαθείς("after arranging"). The story is also told in Apollodoros 3.5.1 with some similarities of wording, including the phrase καθαρθεὶς ὑπὸ Ῥέας καὶ τὰς τελετὰς ἐκμαθών καὶ λαβὼν παρ᾽ἐκείνης τὴν στολήν ("After being purified by Rhea and after learning the rites and receiving from her the equipment"). Apollodoros' ἐκμαθών is a closer match for Z's διδαχθεὶς, but the Venetus A clearly reads διαθείς.]

**Comment by GKolov** [comment]
Yes, Casey!  There are also some other  (big) passages which have errors in the Greek  transcription,  and consequently it's difficult to translate them correctly.  Yes, it will be useful to make a list  of them.

**Comment by caseydue** [comment]
@neelsmith and @Eumaeus, the Greek text of this scholion needs correction in our data. In the place where I have put brackets - Διόνυσος ὁ Διος [] Κυβέλοις τῆς Φρυγιας - the words καὶ Σεμέλης παῖς ἐν are missing. It should read: Διόνυσος ὁ Διος καὶ Σεμέλης παῖς ἐν Κυβέλοις τῆς Φρυγιας...
